### PR TITLE
Update batch processing length normalization to match non-batch processing length normalization

### PR DESCRIPTION
**Pull request by bound-to-love**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -2070,9 +2070,9 @@ void usageBus() {
        << "    --rf-stranded             Strand specific reads for UMI-tagged reads, first read reverse" << endl
        << "    --unstranded              Treat all read as non-strand-specific" << endl
        << "    --paired                  Treat reads as paired" << endl
-       << "    --long                  	Treat reads as long" << endl
+       << "    --long                    Treat reads as long" << endl
        //<< "    --error_rate              Estimated error rate of long reads (required for --long)" << endl
-       << "    --threshold		            Threshold for rate of unmapped kmers per read" << endl
```
Comment (mschilli87) [inline src/main.cpp:2075]: This looks accidental.

File: src/main.cpp [hunk 2]
```diff
@@ -2180,7 +2180,7 @@ void usageTCCQuant(bool valid_input = true) {
        << "                              (default: equivalence classes are taken from the index)" << endl
        << "-f, --fragment-file=FILE      File containing fragment length distribution" << endl
        << "                              (default: effective length normalization is not performed)" << endl
-       << "--long			                    Use version of EM for long reads " << endl
```
Comment (mschilli87) [inline src/main.cpp:2183]: See above.

File: src/main.cpp [hunk 3]
```diff
@@ -2380,7 +2380,7 @@ int main(int argc, char *argv[]) {
               	         if (fld_lr_c[i] > 0.5) {
 		             //Good results with comment below. 
 		             //flensout_f << std::fabs((double)fld_lr[i] / (double)fld_lr_c[i] - index.k);//index.target_lens_[i] - (double)fld_lr[i] / (double)fld_lr_c[i] - k); // take mean of recorded uniquely aligning read lengths 
- 		             flensout_f << std::fabs(index.target_lens_[i] - ((double)fld_lr[i] / (double)fld_lr_c[i]) - index.k);
```
Comment (mschilli87) [inline src/main.cpp:2383]: Care to elaborate a bit? Ideally in comment, else maybe at least in the commit message?

**Review by mschilli87 — CHANGES_REQUESTED** [review]
I had a quick look and thinks this needs work.

File: src/main.cpp (hunk 3 above)
Comment (bound-to-love) [reply src/main.cpp:2383]: Hi! Based on our analysis for effective length normalization for long reads, the updated effective length provides better results.

File: src/main.cpp (hunk 2 above)
Comment (bound-to-love) [reply src/main.cpp:2183]: Could you clarify your question?

File: src/main.cpp (hunk 1 above)
Comment (bound-to-love) [reply src/main.cpp:2075]: Could you clarify what looks accidental?

File: src/main.cpp (hunk 3 above)
Comment (mschilli87) [reply src/main.cpp:2383]: I still don't follow what you are changing and why but I am not familiar with the code. So if this makes sense to others without further explanation, feel free to ignore my comment.

File: src/main.cpp (hunk 2 above)
Comment (mschilli87) [reply src/main.cpp:2183]: The same problem as in line 2075.

File: src/main.cpp (hunk 1 above)
Comment (mschilli87) [reply src/main.cpp:2075]: According to the diff on github, you are removing whitespace here:
image
In line 2073 you fix the alignment of 'Treat' but in line 2075, 'Threshold' is shifted to left too much.

**Review by mschilli87 — CHANGES_REQUESTED** [review]
@bound-to-love: I tried to answer your questions.

**Pull request merged**